spectractor/parameters.py

```diff
@@ -134,8 +134,8 @@ def __getattr__(name):
 
 # Plots
 DISPLAY = True
-if os.environ.get('DISPLAY', '') == '':
-    mpl.use('agg')
+backend = mpl.get_backend()
+if os.environ.get('DISPLAY', '') == '' or backend.lower() in ["agg", "pdf", "ps", "svg", "pgf", "cairo"]:
```

Comment:
This fixes [DM-45832](https://rubinobs.atlassian.net/browse/DM-45832)?

Reply:
Ah, I didn't realize there was a ticket for that!  Yes, I believe it should fix that bug.